[2024-12-24T00:52:49.546+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-24T00:52:49.617+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:52:44.144588+00:00 [queued]>
[2024-12-24T00:52:49.652+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:52:44.144588+00:00 [queued]>
[2024-12-24T00:52:49.653+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-12-24T00:52:49.693+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): step_create_original_table> on 2024-12-24 00:52:44.144588+00:00
[2024-12-24T00:52:49.791+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'NY_Pipeline_ETL_and_ML_Training', 'step_create_original_table', 'manual__2024-12-24T00:52:44.144588+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl_train_pipeline.py', '--cfg-path', '/tmp/tmpwph2cjbt']
[2024-12-24T00:52:49.820+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask step_create_original_table
[2024-12-24T00:52:49.863+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=187) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-24T00:52:49.865+0000] {standard_task_runner.py:72} INFO - Started process 195 to run task
[2024-12-24T00:52:50.203+0000] {task_command.py:467} INFO - Running <TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:52:44.144588+00:00 [running]> on host 9c7b8f26d27a
[2024-12-24T00:52:50.711+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Artur Tiburski' AIRFLOW_CTX_DAG_ID='NY_Pipeline_ETL_and_ML_Training' AIRFLOW_CTX_TASK_ID='step_create_original_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-24T00:52:44.144588+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-24T00:52:44.144588+00:00'
[2024-12-24T00:52:50.725+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-24T00:54:00.642+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to None. Terminating instance.
[2024-12-24T00:54:00.645+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-24T00:54:00.650+0000] {process_utils.py:132} INFO - Sending 15 to group 195. PIDs of all processes in the group: [195]
[2024-12-24T00:54:00.651+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 195
[2024-12-24T00:55:00.669+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=195, name='airflow task ru', status='sleeping', started='00:52:49') did not respond to SIGTERM. Trying SIGKILL
[2024-12-24T00:55:00.671+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 195
[2024-12-24T00:55:00.699+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=195, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='00:52:49') (195) terminated with exit code -9
[2024-12-24T00:55:00.700+0000] {standard_task_runner.py:190} ERROR - ('Job 4 was killed before it finished (likely due to running out of memory)', 'For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed')
